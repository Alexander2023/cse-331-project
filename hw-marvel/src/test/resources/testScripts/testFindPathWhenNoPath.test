# Tests that path finding works properly when there is no path between existing nodes

CreateGraph g

AddNode g c1

AddNode g c2

FindPath g c1 c2