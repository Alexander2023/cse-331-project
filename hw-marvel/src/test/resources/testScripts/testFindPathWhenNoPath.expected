# Tests that path finding works properly when there is no path between existing nodes

created graph g

added node c1 to g

added node c2 to g

path from c1 to c2:
no path found